substring/main.c

```diff
@@ -0,0 +1,38 @@
+#include "findSubstring.h"
+#include "readTextFromFile.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main()
+{
+    int testFailed = test();
```

Comment:
warning: implicit declaration of function ‘test’ [-Wimplicit-function-declaration]